kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/NMDbusConnector.java

```diff
@@ -474,4 +543,103 @@ private void configurationEnforcementDisable() throws DBusException {
         }
         this.configurationEnforcementHandlerIsArmed = false;
     }
+
+    private String getDeviceIdFromNM(String devicePath) throws DBusException {
+        Properties nmModemProperties = this.dbusConnection.getRemoteObject(NM_BUS_NAME, devicePath,
+                Properties.class);
+        String deviceId = (String) nmModemProperties.Get(NM_DEVICE_BUS_NAME + ".Modem", "DeviceId");
+        logger.debug("Found DeviceId {} for device {}", deviceId, devicePath);
+        return deviceId;
+    }
+
+    private Map<DBusPath, Map<String, Map<String, Variant<?>>>> getManagedObjects() throws DBusException {
```

Comment:
I would add a `MM` for making it obvious that this refers to ModemManager. 

```suggestion
    private Map<DBusPath, Map<String, Map<String, Variant<?>>>> getMMManagedObjects() throws DBusException {
```

I think that shortly I'll need to perform the same operation for wpa_supplicant and a name clash is likely